packages/payload/src/translations/ar.json

```diff
@@ -69,6 +69,8 @@
     "invalidFileType": "نوع ملف غير صالح",
     "invalidFileTypeValue": "نوع ملف غير صالح: {{value}}",
     "loadingDocument": "حدثت مشكلة أثناء تحميل المستند برقم التعريف {{id}}.",
+    "localesNotSaved_one": "لم يتم حفظ اللغة التالية:",
```

Comment:
The `localesNotSaved` changes are from generating missing language and unrelated to this change, correct?

Reply:
@DanRibbens Correct.